### PR TITLE
Remove genomelake

### PR DESCRIPTION
**Pull request by haimasree**
genomelake removed as discussed. I have used your one_hot_encode_sequence and the extractors from genomelake and put them in a separate file for future reference. I changed test_github_permalink to include a model which does not have genomelake as dependency. I would get the following error otherwise - 
```
module genomelake not found
```
Is that okay? 

I am keeping this pr open - we can discuss on monday.

### CODE REVIEW

File: docs/templates/contributing/04_Writing_dataloader.py.md [hunk 1]
```diff
@@ -85,9 +85,9 @@ Here is an example dataloader that gets as input a [fasta](http://genetics.bwh.h
 from __future__ import absolute_import, division, print_function
 import numpy as np
 from pybedtools import BedTool
-from genomelake.extractors import FastaExtractor
 from kipoi.data import Dataset
 from kipoi.metadata import GenomicRanges
+from kipoi.helper import FastaExtractor
 
 class SeqDataset(Dataset):
```
Comment (Avsecz) [inline docs/templates/contributing/04_Writing_dataloader.py.md:92]: Could be replaced by kipoiseq.FastaStringExtractor followed by kipoiseq.transforms.one_hot_dna

File: example/models/extended_coda/dataloader.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import numpy as np
 import pybedtools
 
-from genomelake.extractors import ArrayExtractor
+from kipoi.helper import ArrayExtractor
```
Comment (Avsecz) [inline example/models/extended_coda/dataloader.py:7]: This test could be removed

File: example/models/non_bedinput_model/dataloader.py [hunk 3]
```diff
@@ -9,13 +9,11 @@
...
 from pybedtools import BedTool
 
-from genomelake.extractors import BaseExtractor, FastaExtractor, one_hot_encode_sequence, NUM_SEQ_CHARS
 from pysam import FastaFile
 from concise.utils.position import extract_landmarks, ALL_LANDMARKS
 
-
 from kipoi.data import Dataset
-
+from kipoi.helper import BaseExtractor, FastaExtractor, NUM_SEQ_CHARS, one_hot_encode_sequence
```
Comment (Avsecz) [inline example/models/non_bedinput_model/dataloader.py:16]: Could copy in BaseExtractor from genomelake here and use FastaStringExtractor from kipoiseq

File: example/models/pyt/dataloader.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 import pybedtools
 from pybedtools import BedTool
-from genomelake.extractors import FastaExtractor
+from kipoi.helper import FastaExtractor
```
Comment (Avsecz) [inline example/models/pyt/dataloader.py:10]: Use FastaStringExtractor from kipoiseq

File: example/models/rbp/dataloader.py [hunk 5]
```diff
@@ -12,13 +12,14 @@
...
 
 from sklearn.preprocessing import FunctionTransformer
-from genomelake.extractors import BaseExtractor, FastaExtractor, one_hot_encode_sequence, NUM_SEQ_CHARS
 from pysam import FastaFile
 from concise.preprocessing.splines import encodeSplines
 from concise.utils.position import extract_landmarks, ALL_LANDMARKS
 from kipoi.metadata import GenomicRanges
 import linecache
 from kipoi.data import Dataset
+from kipoi.helper import BaseExtractor, FastaExtractor, NUM_SEQ_CHARS, one_hot_encode_sequence
```
Comment (Avsecz) [inline example/models/rbp/dataloader.py:21]: See above

File: kipoi/helper.py [hunk 6]
```diff
@@ -0,0 +1,167 @@
+import six
```
Comment (Avsecz) [inline kipoi/helper.py:1]: Please remove this

**Review by Avsecz — CHANGES_REQUESTED** [review]

**Comment by haimasree** [comment]
I have addressed all the review and have moved one_hot_dna to https://github.com/kipoi/kipoiseq/tree/add-one-hot-encode. kipoiseq branch needs to be updated after one_hot_dna is added to kipoiseq's master branch

**Pull request merged**